src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -104,7 +107,7 @@ private function getVariables(array $envFiles): array
         return $output;
     }
 
-    private function getEnvFiles(): array
+    private function getEnvFiles(?string $dotenvDirectory): array
     {
         $files = [
             '.env.local.php',
```

Comment:
Here, all the hardcoded `.env` must be replaced by the value in `dotenv_path`.

Reply:
See how the file names are build in `Dotenv`.
https://github.com/symfony/symfony/blob/cd78e1359c61b59b3541cfae1d46e0997862f11a/src/Symfony/Component/Dotenv/Dotenv.php#L132-L138